features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/tiers/tiers.jag

```diff
@@ -22,19 +24,41 @@ var getTiers = function () {
     }
 };
 
-var getAppTiers = function () {
+var getApplicationTiers = function () {
     var tiers,
-            log = new Log(),
-            store = jagg.module("manager").getAPIStoreObj();
+            log = new Log();
```

Comment:
fixed in commit: https://github.com/ruks/carbon-apimgt/commit/0f41b03ec368c33c381cc33a069b5ac692744050